### DISCUSSION TITLE
虚拟群晖DDNS-GO无法获取IPV6地址

### DISCUSSION THREAD

**Discussion by sun1014** [jeessy2/ddns-go#1228] [Q&A]
用WMware虚拟出一个黑群，在跑DDNS-GO套件时（DOCKER也是一样），无法获得IPV6网卡地址。用网卡方式不成功，用接口和命令方式也不成功。（浏览器访问https://6.ipw.cn/，可以反馈回一个IPV6地址）

我虚拟机网卡配置是桥接模式，没有勾选“复制物理网络连接状态”。IPV4的话网卡可识别。

该如何解决，是不是没有识别网卡，用接口方式和命令方式也无法获取地址？

**Comment by jeessy2** [comment]
虚拟机好像是这样的，不建议在虚拟机中使用ipv6
